src/core/dbus-execute.c

```diff
@@ -689,6 +690,47 @@ static int property_get_input_data(
         return sd_bus_message_append_array(reply, 'y', c->stdin_data, c->stdin_data_size);
 }
 
+static int property_get_restrict_filesystems(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        ExecContext *c = userdata;
+        _cleanup_strv_free_ char **l = NULL;
```

Comment:
This should be `_cleanup_free` otherwise you'll free the set keys, those are not copied when calling `set_get_strv`.

Reply:
Ah, that makes sense. Great catch! Thank you!